### PR TITLE
Template changes 202403.3.0

### PR DESCRIPTION
**Pull request by srajeshwari-conga**
End to end testing has been done.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -27,6 +27,9 @@ export const config$: Observable<AuthOptions> = ajax.getJSON(configUrl).pipe(
 
 config$.pipe(take(1)).subscribe((authOptions: AuthOptions) => {
   environment.endpoint = authOptions.apiEndpoint;
+  environment.endpoint = authOptions.apiEndpoint;
```
Comment (dboppana-conga) [inline src/main.ts:30]: Remove duplicate statement

**Comment by vbhan-conga** [comment]
@srajeshwari-conga please resolve merge conflicts.

File: src/main.ts (hunk 1 above)
Comment (srajeshwari-conga) [reply src/main.ts]: Changes added.

File: src/app/modules/checkout/layout/checkout.component.html [hunk 2]
```diff
@@ -23,6 +23,22 @@ <h4 class="mb-3 text-muted">{{'CART.CREATE_ORDER' | translate}}</h4>
         </div>
       </div>
     </div>
+    <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between"> <!-- TODO: Added for demo purpose. Need to revisit with UX team-->
```
Comment (vbhan-conga) [inline src/app/modules/checkout/layout/checkout.component.html:26]: We shouldn't push TODO comments like these to public repo, please take care of this.

File: src/app/modules/quotes/quote-create/create-quote.component.html [hunk 3]
```diff
@@ -23,6 +23,20 @@ <h4 *ngIf="!cart?.Proposald?.Id">{{'COMMON.REQUEST_QUOTE' | translate }}</h4>
                     </button>
                 </div>
             </div>
+            <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between"> <!-- TODO: Added for demo purpose. Need to revisit with UX team-->
```
Comment (vbhan-conga) [inline src/app/modules/quotes/quote-create/create-quote.component.html:26]: Same as https://github.com/congarevenuecloud/partner-commerce/pull/11/files#r1599474895

**Review by vbhan-conga — APPROVED** [review]

**Pull request merged**